src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultFunction = (a, b) => {
+    const prevElement = String(a);
+    const currentElement = String(b);
+
+    if (currentElement > prevElement) {
+      return -1;
+    } else if (currentElement < prevElement) {
+      return 1;
+    } else {
+      return 0;
+    }
```

Comment:
```suggestion
    if (currentElement > prevElement) {
      return -1;
    }
    
    if (currentElement < prevElement) {
      return 1;
    }
    
    return 0;
```
There is no need to use else...if here, as your code will not go further after return anyway